apps/alchimie/alchimie/assets.py

```
@@ -1 +1,18 @@
+import dagster as dg
 
+
+@dg.asset
+def my_asset():
```

Ruff (0.8.2)

Missing return type annotation for public function `my_asset`

Add return type annotation: `str`

(ANN201)